cypress/local-only/support/commands.ts

```diff
@@ -112,6 +115,22 @@ Cypress.Commands.add('waitUntilSettled', (maxTries = 20) => {
   waitAndSee(0);
 });
 
+Cypress.Commands.add('runA11y', () => {
```

Comment:
Rather than injecting this into the global cypress can this be extracted and used like a function?

I typically want to avoid adding anything to the cypress object as it makes it harder to trace and has more complex syntax than a typical Javascript function. https://docs.cypress.io/api/cypress-api/custom-commands#Best-Practices